packages/@aws-cdk/aws-ecs-patterns/lib/ecs/application-load-balanced-ecs-service.ts

```diff
@@ -16,6 +17,15 @@ export interface ApplicationLoadBalancedEc2ServiceProps extends ApplicationLoadB
    */
   readonly taskDefinition?: Ec2TaskDefinition;
 
+  /**
+   * The subnets to associate with the service.
+   *
+   * This property is only used for tasks that use the awsvpc network mode.
+   *
+   * @default - Public subnets if `assignPublicIp` is set, otherwise the first available one of Private, Isolated, Public, in that order.
+   */
+  readonly serviceVpcSubnets?: SubnetSelection;
```

Comment:
is there a reason why we don't name this as `vpcSubnets`? I don't see any definition for `vpcSubnets` currently.

Reply:
Also can we move this field to base service file? I think in that way we don't need to define this field separately in ecs fargate and fargate pattern.

Reply:
Hi, please excuse my late reply. the reason why i named it `serviceVpcSubnets` is to make more clear that it is meant for the service, as a seperate configuration can be made for the load balancer.

the reason why i didnt add it to the base service file is that i adapted it from other files and there is a slight change in doc comments